src/styles/examples.scss

```diff
@@ -111,7 +111,6 @@ article {
   cursor: pointer;
   font-family: Sintony, Helvetica, sans-serif;
   font-size: 14px;
-  outline: none;
```

Comment:
To restore focus styling to the button - better option would be to explicitly define custom focus styling - I'll do that